<launch>
    <arg name="config_yaml" />

    <!--
    FIRST POS: office
    SECOND: house
    THIRD: condo_floor
    -->

    <!--robot 1-->
    <group ns="robot_1">
        <include file="$(find learning_toponav)/launch/includes/one_robot.launch">
            <arg name="robot_name" value="robot_1" />
            <arg name="color" value="'1 0 0 1'" /> <!--red-->
            <arg name="config_yaml" value="$(arg config_yaml)" />

            <!--<arg name="x_pos" value="4" /><arg name="y_pos" value="4" />-->
            <!--<arg name="x_pos" value="-3" /><arg name="y_pos" value="1" />-->
            <arg name="x_pos" value="-1" /><arg name="y_pos" value="3" />
            <arg name="z_pos" value="0" />
        </include>
    </group>

    <!--robot 2-->
    <group ns="robot_2">
        <include file="$(find learning_toponav)/launch/includes/one_robot.launch">
            <arg name="robot_name" value="robot_2" />
            <arg name="color" value="'0 0 1 1'" /> <!--blue-->
            <arg name="config_yaml" value="$(arg config_yaml)" />

            <!--<arg name="x_pos" value="-2" /><arg name="y_pos" value="4" />-->
            <!--<arg name="x_pos" value="-6" /><arg name="y_pos" value="3" />-->
            <arg name="x_pos" value="1" /><arg name="y_pos" value="3" />
            <arg name="z_pos" value="0" />
        </include>
    </group>

    <!--robot 3-->
    <group ns="robot_3">
        <include file="$(find learning_toponav)/launch/includes/one_robot.launch">
            <arg name="robot_name" value="robot_3" />
            <arg name="color" value="'0.9 0.9 0 1'" />  <!--yellow-->
            <arg name="config_yaml" value="$(arg config_yaml)" />

            <!--<arg name="x_pos" value="-2" /><arg name="y_pos" value="-1" />-->
            <!--<arg name="x_pos" value="4" /><arg name="y_pos" value="1" />-->
            <arg name="x_pos" value="6" /><arg name="y_pos" value="3" />
            <arg name="z_pos" value="0" />
        </include>
    </group>

    <!--robot 4-->
    <group ns="robot_4">
        <include file="$(find learning_toponav)/launch/includes/one_robot.launch">
            <arg name="robot_name" value="robot_4" />
            <arg name="color" value="'0 1 0 1'" /> <!--green-->
            <arg name="config_yaml" value="$(arg config_yaml)" />

            <!--<arg name="x_pos" value="3" /><arg name="y_pos" value="-1" />-->
            <!--<arg name="x_pos" value="-6" /><arg name="y_pos" value="-1" />-->
            <arg name="x_pos" value="4" /><arg name="y_pos" value="-3" />
            <arg name="z_pos" value="0" />
        </include>
    </group>

    <!--robot 5-->
    <group ns="robot_5">
        <include file="$(find learning_toponav)/launch/includes/one_robot.launch">
            <arg name="robot_name" value="robot_5" />
            <arg name="color" value="'1 0 1 1'" /> <!--purple-->
            <arg name="config_yaml" value="$(arg config_yaml)" />

            <!--<arg name="x_pos" value="3" /><arg name="y_pos" value="2" />-->
            <!--<arg name="x_pos" value="5" /><arg name="y_pos" value="-3" />-->
            <arg name="x_pos" value="0" /><arg name="y_pos" value="-3" />
            <arg name="z_pos" value="0" />
        </include>
    </group>

    <!--robot 6-->
    <group ns="robot_6">
        <include file="$(find learning_toponav)/launch/includes/one_robot.launch">
            <arg name="robot_name" value="robot_6" />
            <arg name="color" value="'1 0.6 0 1'" /> <!--orange-->
            <arg name="config_yaml" value="$(arg config_yaml)" />

            <!--<arg name="x_pos" value="0" /><arg name="y_pos" value="4" />-->
            <!--<arg name="x_pos" value="1" /><arg name="y_pos" value="3" />-->
            <arg name="x_pos" value="-3" /><arg name="y_pos" value="-3" />
            <arg name="z_pos" value="0" />
        </include>
    </group>

    <!--robot 7-->
    <group ns="robot_7">
        <include file="$(find learning_toponav)/launch/includes/one_robot.launch">
            <arg name="robot_name" value="robot_7" />
            <arg name="color" value="'0 0.8 1 1'" /> <!--cyan-->
            <arg name="config_yaml" value="$(arg config_yaml)" />

            <!--<arg name="x_pos" value="-5" /><arg name="y_pos" value="0" />-->
            <!--<arg name="x_pos" value="-1" /><arg name="y_pos" value="3" />-->
            <arg name="x_pos" value="-7" /><arg name="y_pos" value="-2" />
            <arg name="z_pos" value="0" />
        </include>
    </group>


    <!--robot 8-->
    <!--<group ns="robot_8">-->
    <!--    <include file="$(find learning_toponav)/launch/includes/one_robot.launch">-->
    <!--        <arg name="robot_name" value="robot_8" />-->
    <!--        <arg name="color" value="'1 0 0.5 1'" /> &lt;!&ndash;pink&ndash;&gt;-->
    <!--        <arg name="config_yaml" value="$(arg config_yaml)" />-->

    <!--        &lt;!&ndash;<arg name="x_pos" value="7" /><arg name="y_pos" value="-1" />&ndash;&gt;-->
    <!--        &lt;!&ndash;<arg name="x_pos" value="-3" /><arg name="y_pos" value="1" />&ndash;&gt; &lt;!&ndash;CHANGE ! &ndash;&gt;-->
    <!--        <arg name="x_pos" value="5" /><arg name="y_pos" value="0" />-->
    <!--        <arg name="z_pos" value="0" />-->
    <!--    </include>-->
    <!--</group>-->
</launch>